mgmt/rest/metric.go

```diff
@@ -40,6 +41,7 @@ func (s *Server) getMetrics(w http.ResponseWriter, r *http.Request, _ httprouter
 	// perform a query
 	q := r.URL.Query()
 	v := q.Get("ver")
+	apiVersion := r.URL.Path[1:3]
```

Comment:
I'll add a comment to address that. If it gets something back other than "v2" it will default to v1. I think that will be okay.